TheWarWithin/ShamanRestoration.lua

```diff
@@ -955,6 +966,7 @@ spec:RegisterAbilities( {
         cooldown = function() return talent.call_of_the_elements.enabled and 120 or 180 end,
         gcd = "spell",
         school = "nature",
+        toggle = "cooldowns",
```

Comment:
I think I normally gate `totemic_recall` behind the `interrupts` toggle since that's supposed to be Interrupts/Utility, but I'm fine with this.

Would we use different logic for Totemic Recall depending on whether we are in healing mode or not?  If so, we can specify that it's on CDs only if we're healing (or vice versa):

`toggle = function() if settings.healing_mode then return "cooldowns" end,`

In the above case, if not in healing mode, then the default toggle will be Cooldowns, otherwise it won't be on a toggle.  This can still be overridden in the Abilities section.

Reply:
Oh neat, I didn't think of that. I like it.

Reply:
There is definitely use cases outside of pure healing, such as reseting CC/utility totems